### ISSUE TITLE
Tests updates in staging and add secrets for production

### ISSUE THREAD

**Issue by dlapiduz** [18F/cg-deploy-docker-swarm#13]
We need to make sure we understand what happens to all the boxes when we change a stemcell or upgrade the docker version.

Once that is done we need to add the secrets for production and deploy there.

**Comment by mogul** [comment]
Likely to be superseded by upcoming service reliability work (which will involve a move to another solution).
